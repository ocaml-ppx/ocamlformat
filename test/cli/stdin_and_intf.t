  $ echo 'val x :     int' | ocamlformat --intf -
  val x : int
