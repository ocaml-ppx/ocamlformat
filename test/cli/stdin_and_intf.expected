val x : string

module X : S
(** Make sure this file cannot be parsed as an impl file *)
